CGPs/cgp-0158/mainnet.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+      "contract": "GoldToken",
+      "address": "0x471EcE3750Da237f93B8E339c536989b8978a438",
+      "function": "Approve",
+      "args": [
+        "0xd71efa410B6EaAB0bAc3b515B393C886BE70F09E",
+        "3838900000000000"
+      ],
+      "value": "0"
+    },
+    {
+      "contract": "StableToken",
+      "address": "0x765DE816845861e75A25fCA122bb6898B8B1282a", 
+      "function": "Approve",
+      "args": [
+        "0xd71efa410B6EaAB0bAc3b515B393C886BE70F09E",
+        "3000000000000000"
```

Comment:
The amounts are represented in gwei when the protocol expects them in wei.

Where `3000000000000000` gwei should be `3000000000000000000000000` wei

Reply:
fixed